doc/usecases/elektra_web/UC_finding_keys.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: Finding keys
+
+## Summary
+
+Title: Finding keys
+Scope: Configuration
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User searches for a key by filtering the key database.
+
+## Scenarios
+
+Precondition: View configuration of an instance
+Main success scenario: User searches for a key by entering information (its path
+  or data) in a field. The whole key database is filtered according to the
+  contents of the search field.
```

Comment:
As described, the search field works like a filter, so all keys that match the search input will be shown, the other keys will be hidden. I described this more clearly now.